sourced/ml/transformers/basic.py

```diff
@@ -34,6 +34,13 @@ def __call__(self, head):
             self.trace = self.path()
         return self.head
 
+    @staticmethod
+    def maybe(pipeline, persistence):
```

Comment:
hehe) maybe rename it to `set_persistence`?

Reply:
It says `Cacher.maybe`. Rather funny and you already remember it)